pkgs/by-name/ru/rutabaga-gfx-ffi/package.nix

```diff
@@ -0,0 +1,49 @@
+{ aemu
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, gfxstream
+, libdrm
+, lib
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "rutabaga_gfx_ffi";
+  version = "unstable-2023-12-20";
```

Comment:
Why not the released version?

Reply:
Without my patches that target current master, it's basically unusable.